cmd/pebble/cmd_plan_test.go

```diff
@@ -25,16 +25,27 @@ import (
 )
 
 func (s *PebbleSuite) TestGetPlan(c *check.C) {
+	fail := false
+
 	s.RedirectClientToTestServer(func(w http.ResponseWriter, r *http.Request) {
 		c.Check(r.Method, check.Equals, "GET")
 		c.Check(r.URL.Path, check.Equals, "/v1/plan")
 		c.Check(r.URL.Query(), check.DeepEquals, url.Values{"format": []string{"yaml"}})
-		fmt.Fprint(w, `{
+
+		if fail {
```

Comment:
This is one of those that I'd prefer to avoid the flag and just add another test (even if it's a few more lines of code). It's simpler to reason about, and avoids the need for `s.ResetStdStreams()`.